github/event_types.go

```diff
@@ -268,6 +268,22 @@ type LabelEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// MarketplacePurchaseEvent is triggered when a user purchases, cancels, or changes
+// their GitHub Marketplace plan.
+// Webhook event name "marketplace_purchase".
+//
+// Github API docs: https://developer.github.com/v3/activity/events/types/#marketplacepurchaseevent
+type MarketplacePurchaseEvent struct {
+	Action *string `json:"action,omitempty"`
```

Comment:
It'd be nice to add a comment here documenting possible action values, like in the events above and below.

```
// Action is the action that was performed. Possible values are:
// "purchased", "cancelled", "changed".
```